packages/x-components/src/x-modules/history-queries/components/my-history.vue

```diff
@@ -1,58 +1,78 @@
 <template>
-  <BaseSuggestions
-    :suggestions="historyQueries"
-    class="x-my-history-queries"
-    data-test="my-history-queries"
-    :animation="animation"
-  >
-    <template #default="{ suggestion, index }">
-      <!--
+  <ul v-if="historyQueries.length > 0" class="x-my-history x-list">
+    <li
+      v-for="(list, date) in historyQueriesGroupByDate"
+      :key="date"
+      class="x-my-history-item"
+      data-test="my-history-item"
+    >
+      <span class="x-my-history-item__date" data-test="my-history-date">{{ date }}</span>
```

Comment:
Add a `slot` to customize the date:

```suggestion
      <slot name="date" :date="date">
        <span class="x-my-history-item__date" data-test="my-history-date">{{ date }}</span>
      </slot>
```